script/Ciel/Day 9/09_01_CIEL09_8.txt

```diff
@@ -987,6 +987,7 @@ Something like that would never be enough to change the fact that I care deeply
 [sha:67dee764df33b5affd0828d9bad40ef3316f63ca]{
 -- Page 761, Offset 24114.
 -- ただの顔見知りとか、それぐらいの。そうすれば貴方の方から話しかけてくる事も、こんな風に、嫌われる事もなかった。
+// Flag
 I wanted us to just be acquaintances, or something like that.
  That way, you wouldn't have been the one to strike up a conversation.
  Things wouldn't have ended with you hating me.
```

Comment:
> It's a bit long.
> 
> I wanted us to just be acquaintances, or something like that.
> That way, you wouldn't have come up to speak with me all the time.
> And then, you wouldn't have ended up hating me.
> 
> If we use "And then" on the second, it becomes apparent that the two are connected, without it needing to be so unnaturally elaborated.

Found my old comment. Anyway, its the connectivity between the "That way, you wouldn't have been the one to strike up a conversation." and "Things wouldn't have ended with you hating me." that really bothers me. That and the way the second sentence is worded in general.

Maybe add a "like this" at the end of the last sentence.